src/Test/Perf/runner.csx

```diff
@@ -9,40 +10,64 @@ using System;
 
 InitUtilities();
 
-var myDir = MyWorkingDirectory();
-var skip = new HashSet<string> {
-    Path.Combine(myDir, "runner.csx"),
-    Path.Combine(myDir, "bootstrap.csx"),
-    Path.Combine(myDir, "util"),
-    Path.Combine(myDir, "infra"),
-};
+var testDirectory = PerfTestDirectory();
 
 var allResults = new List<Tuple<string, List<Tuple<int, string, object>>>>();
 var failed = false;
 
+// Run DownloadCPC before using the TraceManager because TraceManager uses the CPC downloaded binaries 
+DownloadCPC();
```

Comment:
Should this be part of bootstrap.bat?  (Does it need to happen on each run, or is it good enough to do it once per machine?)